Source/Orts.Simulation/Simulation/AIs/AITrain.cs

```diff
@@ -702,9 +702,8 @@ public void AIUpdate(float elapsedClockSeconds, double clockTime, bool preUpdate
                 }
 
                 // set wipers on or off
-                try
-                {
-                    MSTSLocomotive leadingloco = Cars[0] as MSTSLocomotive;
+                MSTSLocomotive leadingloco = Cars[0] as MSTSLocomotive;
+                if (leadingloco != null)
```

Comment:
This change was made by #350, so it also needs to be removed from this PR.

Reply:
Hi Ryan,

Thanks for this.

Not sure of an easy way to undo just this change on GitHub, so altered my source file and pushed a new commit.

Best wishes,

Chris